NYCFireWire/NYCFireWire/src/ViewControllers/DashboardViewController.swift

```diff
@@ -246,16 +251,18 @@ extension DashboardViewController: UITableViewDataSource {
             //Incident
             let cell = tableView.dequeueReusableCell(withIdentifier: "IncidentCell", for: indexPath) as! IncidentTableViewCell
 //            let incident = allIncidents[indexPath.row]
+            cell.isLiked = incident.isLiked
             cell.boroLabel.text = incident.boro
+            cell.dateLabel.text = incident.createdAt.smartStringFromDate()
             if let boxNumber = Int(incident.boxNumber) {
                 cell.boxLabel.text = "Box \(boxNumber)"
             } else {
                 cell.boxLabel.text = incident.boxNumber
             }
             cell.titleLabel.text = "*\(incident.title)*"
             cell.subtitleLabel.text = incident.subtitle
-            cell.addressLabel.text = incident.address
-            cell.timeLabel.text = incident.createdAt.smartStringFromDate()
+            cell.numberOfLikesLabel.text = "\(incident.numberOfLikes) likes"
```

Comment:
If likes is 0, dont show '0 likes' just show thumbs up
If likes is 1, should say 1 like
If likes is greater than 1, say <number> likes